src/jsx-runtime/index.ts

```diff
@@ -100,7 +100,6 @@ jsx.renderDOM = (
   // classes
   if (props['className'] !== undefined) {
     props['class'] = props['className']
```

Comment:
aren't the classes already added thanks to this assigment?

Reply:
No, it doesn't work in firefox in any case, so I kept this in case it worked for other browsers, but the API on firefox uses `className` to set the class for an element

Reply:
we should remove then this assignment, right?